README.md

```diff
@@ -251,7 +251,7 @@
 * Underscore
 
   ```javascript
-  _.isFinite(object)
+  _.isNumber(object) && _.isFinite(object)
```

Comment:
Here's where it get's interesting. Lodash supports the `Number.isNumber` behavior in `_.isFinite` :P